src/Schema/Builder.php

```diff
@@ -13,16 +13,37 @@
 
 class Builder extends \Illuminate\Database\Schema\Builder
 {
-    /** @inheritdoc */
-    public function hasColumn($table, $column)
+    /**
+     * Check if column exists in the collection schema.
+     *
+     * @param $table
+     * @param $column
+     * @return bool
+     */
+    public function hasColumn($table, $column): bool
     {
-        return true;
```

Comment:
Technically, that's a breaking change as we change the return value of this method. MongoDB doesn't have columns and there is no restriction on field names (unless you use a schema validation). I'm understand that this method is not useful as it is, as it returns always `true`, but I don't think the proposed change is correct as it depends on the data in the collection. Which can lead to a different results depending on the data stored.